plugins/sigma.renderers.linkurious/README.md

```diff
@@ -16,6 +16,8 @@ This plugin overrides default renderers to provide more powerful renderers.Nodes
 - `icon`: *object*
 - `image`: *object*
 - `level`: *number*
+- 'border_size': *number*
+- 'border_color': *string*
```

Comment:
I thought that within nodes the custom is to use underscores? e.g. line # 153 refers to the active color as `node.active_color` - whence I decided for the underscores (originally I had camel case).  the code fragment:

```
if (node.active) {
      // Color:
      if (settings('nodeActiveColor') === 'node') {
        color = node.active_color || color;
      }
```